CRM/Core/Form/EntityFormTrait.php

```diff
@@ -221,4 +222,21 @@ protected function isDeleteContext() {
     return ($this->_action & CRM_Core_Action::DELETE);
   }
 
+  protected function setEntityFieldsMetadata() {
+    foreach ($this->entityFields as $field => &$props) {
+      if ($props['not-auto-addable']) {
```

Comment:
This may need to change to 

```suggestion
      if (!empty($props['not-auto-addable'])) {
```

Reply:
Yes. Done :-)